### PR TITLE
Block number

### PR DESCRIPTION
**Pull request by renaynay**
Adds support for `WB GET BLOCK NUMBER` to output the different block number of every node from the CLI.

Reference ticket: https://whiteblockio.atlassian.net/browse/GEN-198

### CODE REVIEW

File: whiteblock/cmd/get.go [hunk 1]
```diff
@@ -304,23 +307,64 @@ func getBlockCobra(cmd *cobra.Command, args []string) {
...
+		wg.Wait()
+
+		util.Print(blockHeights)
+
+		return
+	}
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
```
Comment (n8wb) [inline whiteblock/cmd/get.go]: clean this up

Comment (n8wb) [reply whiteblock/cmd/get.go]: you don't need all this here

**Review by n8wb — CHANGES_REQUESTED** [review]

File: whiteblock/cmd/get.go (hunk 1 above)
Comment (renaynay) [reply whiteblock/cmd/get.go]: I don't need wg.Add and wg.Done?

File: whiteblock/cmd/get.go [hunk 2]
```diff
@@ -304,22 +307,73 @@ func getBlockCobra(cmd *cobra.Command, args []string) {
 	}*/
 }
 
+func getBlockHeightByNode(cmd *cobra.Command, args []string) {
+	wg := sync.WaitGroup{}
+	mux := sync.Mutex{}
+
+	util.CheckArguments(cmd, args, 0, 1)
+
+	if args[0] == "all" {
```
Comment (n8wb) [inline whiteblock/cmd/get.go]: It should be `wb get block number --all` not wb get block number all, also this will panic. Switch to using a flag.

Comment (renaynay) [reply whiteblock/cmd/get.go]: I am using a flag

**Review by n8wb — APPROVED** [review]

**Pull request merged**